src/main/java/org/spring/FullVersion/Human.java

```diff
@@ -53,8 +56,30 @@ public Human(String name, String surname, String birthDateString, int iq) {
         this.birthDate = convertToMillis(birthDateString);
     }
 
-    private long convertToMillis(String birthDateString) {
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    public Human(String childName, String childSurname, int childYear, int childIq) {
+        this.name = childName;
+        this.surname = childSurname;
+        this.iq = childIq;
+        this.birthDate = childYear;
+
+    }
+    public Human(String name, String surname, long birthDate, int iq) {
+        this.name = name;
+        this.surname = surname;
+        this.birthDate = birthDate;
+        this.iq = iq;
+    }
+
+
+    public Human(String fatherName, String fatherSurname, int fatherYear, int fatherMonth, int fatherDay, int fatherIq) {
```

Comment:
why we need this constructor? Please read hw07 conditions carefully, there should be additional constructor with String birthdate with format dd/MM/yyyy.